src/lib/PWM/waveform_8266.cpp

```diff
@@ -257,33 +257,30 @@ static IRAM_ATTR void timer1Interrupt() {
           uint32_t desired = 0;
           uint32_t *timeToUpdate;
           wvfState.waveformState ^= mask;
-          if (wvfState.waveformState & mask && wave->nextHighLowUs != 0) {
+          if (wvfState.waveformState & mask) {
+            if (i == 16) {
```

Comment:
What is 16?
Magic?

Reply:
It's the pin number GPIO16. Theres no constant we can use for that. This comes from the original waveform code from the Arduino framework code.

Reply:
Can't add a `#define GPIO16 16` above it? With a comment?